ffi/include/wirefilter.h

```diff
@@ -0,0 +1,122 @@
+#ifndef _WIREFILTER_H_
+#define _WIREFILTER_H_
+
+#include <stdlib.h>
+#include <stdint.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef struct wirefilter_Scheme wirefilter_Scheme;
```

Comment:
I would suggest to use more conventional naming for structure, e.g. `wirefilter_scheme_t`

Reply:
Ok i will update all the types. I actually re-used what we already use internally.

Reply:
Would
```C
typedef struct wirefilter_scheme wirefilter_scheme_t;
typedef struct wirefilter_execution_context wirefilter_execution_context_t;
typedef struct wirefilter_filter_ast wirefilter_filter_ast_t;
typedef struct wirefilter_filter wirefilter_filter_t;
```
be ok?

Reply:
lgtm